tests/src/Integration/Engine/AnnotationProcessingTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Engine\AnnotationProcessingTest.php.
+ */
+
+namespace Drupal\Tests\rules\Integration\Engine;
+
+use Drupal\Tests\rules\Integration\RulesIntegrationTestBase;
+use Drupal\Core\Session\SessionManagerInterface;
+
+/**
+ * Test auto saving of variables after Rules execution.
```

Comment:
Nope, I pushed it.  If you view the file with the View button, you'll see I made the fix.  For some reason, though, the diff shows the old state of the file.  Github bug I'm guessing.